frontend/server/src/Controllers/Certificate.php

```diff
@@ -345,16 +345,46 @@ private static function getContestCertificate(string $verification_code): string
         );
     }
 
+    private static function getCourseCertificate(string $verificationCode): ?string {
+        $certificateData = \OmegaUp\DAO\Certificates::getCourseCertificateByVerificationCode(
+            $verificationCode
+        );
+
+        if (is_null($certificateData)) {
+            return null;
+        }
+
+        $translator = \OmegaUp\Translations::getInstance();
+        $title = utf8_decode(
+            $translator->get('certificatePdfCourseTitle')
+        );
+        $identityName = utf8_decode($certificateData['identity_name']);
+        $description = utf8_decode(
+            sprintf(
+                $translator->get('certificatePdfCourseDescription'),
+                $certificateData['course_name']
+            )
+        );
```

Comment:
Esto se nos pasó en revisiones anteriores, una disculpa. 

Teenemos la función `\OmegaUp\ApiUtils::formatString` para hacer la interppolación de cadenas en PHP. Básicamente se envía como primer parámetro la cadena de traducción que vamos a ocupar, y en el segundo parámetro le vamos a enviar un array asociativo con los índices de las cadenas que vamos a usar como datos variables, en este caso, el nombre del curso
```suggestion
        $description = \OmegaUp\ApiUtils::formatString(
            $translator->get('certificatePdfCourseDescription'),
            [
                'course_name' => $certificateData['course_name'],
            ],
            convertUTF8ToISO: true
        );
```
si te fijas, estoy mandando un tercer parámetro que actualmente no existe . Este nos puede servir con el propósito de que todas las cadenas interpoladas que se van a usar en el PDF tambien pasen por la función de `convertUTFToISO`.  Así que en la función de `\OmegaUp\ApiUtils::formatString` habría que agregar ese parámetro y ajustar el código para que haga lo que te explicaba con anterioridad.